pylint/checkers/base/comparison_checker.py

```diff
@@ -343,15 +343,12 @@ def _check_type_x_is_y(
         ):
             return
 
-        if operator in {"is", "is not"} and _is_one_arg_pos_call(right):
+        if operator in {"!=", "==", "is", "is not"} and _is_one_arg_pos_call(right):
```

Comment:
```suggestion
        if _is_one_arg_pos_call(right):
```

the operator check may not be necessary here as it is already performed upstream at `_check_unidiomatic_typecheck`?

Reply:
You're right, nice catch !